tests/test_enterprise/api_client/test_enterprise.py

```diff
@@ -61,39 +60,6 @@ def _assert_num_requests(self, expected_count):
         """
         assert len(responses.calls) == expected_count
 
-    @responses.activate
-    @mock.patch('enterprise.api_client.lms.JwtBuilder', mock.Mock())
-    @mock.patch('enterprise.api_client.discovery.get_edx_api_data', mock.Mock())
-    @mock.patch('enterprise.api_client.discovery.JwtBuilder', mock.Mock())
-    def test_no_response_doesnt_get_cached(self):
```

Comment:
seems like this test is still relevant, and should be updated to check caching on the enterprise catalog endpoint instead.